### PR TITLE
First pass at documentation for adding/changing hero images

### PR DESCRIPTION
**Pull request by TheKitty**
Per Phil. He stated that such documentation should be in markdown on circuitpython.org itself.

In this PR, no accompanying images are used. If images are desired, we will need to determine where to put them. Under /assets or under /doc or somewhere else.

### CODE REVIEW

**Comment by TheKitty** [comment]
I didn't ask GitHub to do anything with the images, must be automatic.....

**Comment by jwcooper** [comment]
It is! And it's ok for this PR I think, it's simply optimizing the file sizes (the new heroes are much smaller).

The issue I think is that normally it doesn't run properly as we push forked PR from our own repositories, so it suddenly has a bunch of images to optimize from previous pull requests. I'll look into that, as I'd like it to optimize images on each PR that comes in with new images.

**Comment by TheKitty** [comment]
Added reviewers as PT stated he would look at it after merge.

File: doc/hero_images.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+### Where the Hero Image is specified in the Site Layout
+
+The hero image on circuitpython.org may be changed by editing a value in the _config.yml file.
+
+The _config.yml file is located in the root directory of the circuitpython.org repo on GitHub.
+
+The typical contents (truncated after a number of lines) is similar to:
+
+  title: CircuitPython
```
Comment (kattni) [inline doc/hero_images.md]: Looks like the formatting here is not proper when it renders - there are no newlines, it renders as a paragraph.

Comment (TheKitty) [reply doc/hero_images.md]: Thanks, yes I thought I had tabs in, changed to 4 spaces and markdown formats right. I also bolded the filenames. Ready again I hope

File: doc/hero_images.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- Go to [https://github.com/adafruit/circuitpython-org/blob/master/_config.yml](https://github.com/adafruit/circuitpython-org/blob/master/_config.yml) 
+and click on the pencil icon on the right side.
+- Go down to find the value "hero_image:"
+- Replace the filename. For example, replace the text CircuitPython_Hero.jpg with the new name of a file placed in /assets/images/heroes, 
+for example Gamer_Hero.png.
+- Scroll to the bottom of the page and put a description of the edit in the box marked "Update _config.yml"
+- Keep the value "Create a new branch and start a pull request" checked.
+- Click the green "Commit changes" button.
+
+This will place you in a new screen where you can describe the changes you have made in the "Leave a ccomment" block.
```
Comment (kattni) [inline doc/hero_images.md]: Should be "leave a comment".

Comment (TheKitty) [reply doc/hero_images.md]: Thank you!! Fixed

**Comment by kattni** [comment]
@TheKitty You'll need to deal with the merge conflicts but otherwise this is ready to merge when you're ready.

**Pull request merged**